Automattic/wp-calypso
# Theme Discovery: Randomized Collections and Theme Cards by config

Related to:
- https://github.com/Automattic/dotcom-forge/issues/4353

The goal of this PR is to provide a mechanism by which we can randomize the order in which the different collections are presented to the users and the themes contained within those collections based on the existence of a feature flag.

## Proposed Changes

* The collections are aware of a feature flag and randomize their content (theme cards) if needed.
* The Discover screen is aware of a feature flag and randomized the collection order if needed.
* Added a feature flag named `themes/discovery-randomize-collections` that indicates weather collections should be randomized or not.
* Added a feature flag named `themes/discovery-randomize-collection-themes` that indicates weather cards within a collection should be randomized or not.
* Set all flags to true in all environments.

Random order:
![Captura de pantalla 2023-10-26 a las 10 53 58](https://github.com/Automattic/wp-calypso/assets/1989914/98389c80-6eb1-441a-bb5d-d89b6e4a5873)


## Testing Instructions



* Open the Calypso Live link
* Navigate to the LoTS
* By default you should always see the same random collections and themes inside those collections, if you reload the order will change.
* If you disable the flags the UI should react accordingly.
* Experiment with both flags on, both off, one on and the other off and vice versa.

## Pre-merge Checklist
